Code/Framework/AzQtComponents/AzQtComponents/Utilities/ScreenGrabber_mac.mm

```diff
@@ -57,8 +57,10 @@
         QRect region({}, m_size);
         region.moveCenter(point);
         CGRect bounds = CGRectMake(region.x(), region.y(), region.width(), region.height());
-
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
         CGImageRef cgImage = CGWindowListCreateImageFromArray(bounds, windows, kCGWindowImageNominalResolution);
+#pragma clang diagnostic pop
         CFRelease(windows);
```

Comment:
Wrap this in a #CARBONATED so we can track it.  Also, please make a GHI for it (and link in this PR) so the O3DE folks can tackle

Reply:
Added CARBONATED fence, created GHI:
https://github.com/o3de/o3de/issues/17928